usr/src/pkg/manifests/system-kernel-platform.p5m

```diff
@@ -198,11 +198,9 @@ $(i386_ONLY)driver name=evtchn perms="* 0666 root sys"
 $(i386_ONLY)driver name=isa class=sysbus alias=pciclass,060100
 $(i386_ONLY)driver name=npe alias=pciex_root_complex
 driver name=ns16550a perms="* 0666 root sys" perms="*,cu 0600 uucp uucp" \
-    alias=ns16550a
+    alias=arm,pl011
 $(i386_ONLY)driver name=pci class=pci
 $(i386_ONLY)driver name=pit_beep alias=SUNW,pit_beep
-driver name=pl011 perms="* 0666 root sys" perms="*,cu 0600 uucp uucp" \
```

Comment:
Do we know this works on qemu too?  Should we just be deleting the pl011 driver etc when we do this, since we won't end up using it anymore?

Reply:
Yes, it works in qemu.

```
root@braich:~# uname -a
SunOS braich 5.11 rpi4-0-gf68c6a6602 armv8 aarch64 QEMU,virt-4.1
root@braich:~# prtconf -dD | grep 165
    pl011 (driver name: ns16550a)
    pl011, instance #1 (driver name: ns16550a)
root@braich:~#
```

I haven't looked at the difference between the drivers but I intend to, and we should hopefully end up deleting one or the other!